### ISSUE TITLE
Can set 'classNameSuffix' but not 'enumNameSuffix' in jsonschema2pojo-maven-plugin

### ISSUE THREAD

**Issue by jasonCodesAway** [joelittlejohn/jsonschema2pojo#852]
Hello,

Are there any plans to add a 'enumNameSuffix' along with the existing 'classNameSuffix' for consistency? The classNameSuffix doesn't affect generated enum names in 1.0.0-alpha2.

Thanks,
Jason

**Comment by joelittlejohn** [comment]
I don't think we need enumNameSuffix, I think we should fix this as a bug (classNameSuffix should apply to enums).
